test/retorts.py

```diff
@@ -100,14 +100,32 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.request_mock.passthru_prefixes = self.passthru_prefixes
 
 
-class AuthResponseHelper(ResponsesHelper):
+class AuthResponseHelper:
```

Comment:
Ignore my earlier comment. But this commit should be squashed with that one. That would have avoided my confusion. And ResponsesHelper should still be deleted.

Reply:
AuthResponseHelper was removed w/ #2969